src/librustdoc/html/static/js/search.js

```diff
@@ -1886,6 +1881,18 @@ class DocSearch {
             (modulePathData.modulePath === "" ?
                 moduleName :
                 `${modulePathData.modulePath}::${moduleName}`);
+        /** @type {function("parent" | "traitParent"): Promise<rustdoc.RowParent>} */
+        const buildParentLike = async field => {
+            const [name, path] = entry !== null && entry[field] !== null ?
+                await Promise.all([this.getName(entry[field]), this.getPathData(entry[field])]) :
+                [null, null];
+            if (name !== null && path !== null) {
+                return { name: name, path: path };
+            } else {
```

Comment:
Also we can remove the `else` condition here since the first branch returned already (it's really nitpicking).

Reply:
No I mean, just the `else`, not the `return` inside the `else`. Please keep this `return`. :wink:

Reply:
ohh, you want me to inline/unwrap the block, not remove the whole branch, yeah that makes a lot more sense

Reply:
Yep yep, sorry for the unclear message. :laughing: